CONTRIBUTING.md

```diff
@@ -58,6 +66,10 @@ Some pointers:
   checks][error-prone-criteria]. Most guidelines described there apply to this
   project as well, except that this project _does_ focus quite heavy on style
   enforcement. But that just makes the previous point doubly important.
+- Make sure that there are no violations of the newly introduced checks in the
+  codebase itself by running the [self check][error-prone-support-self-check].
+  See the [development instructions][error-prone-support-developing] for extra
+  context.
```

Comment:
This mostly repeats the text above. The idea behind referencing the `error-prone-support-developing` was that we don't need to repeat the self check description.